### PR TITLE
fix: covering pagination buttons into 'a' tag

### PR DESCRIPTION
**Pull request by VitaliiDC8**

### CODE REVIEW

**Review by niktverd — CHANGES_REQUESTED** [review]

**Comment by NikitaCG** [comment]
you need to rebase from main, because in your branch have old dependencies, for example https://github.com/gravity-ui/blog-constructor/blob/fix/cover_pagination_into_a_tag/package.json#L82

File: src/components/Paginator/components/PaginatorItem.tsx [hunk 1]
```diff
@@ -18,16 +18,23 @@ export const PaginatorItem = ({
...
+    const navigationLink = `?${mods.type || 'page'}=${itemKey}`;
 
     return (
-        <Button
-            view="flat"
-            size="xl"
-            className={b('item', mods)}
-            onClick={() => onClick?.(itemKey)}
-            loading={loading && Boolean(mods.active)}
+        <a
```
[the remark this answers is not in the document]
Comment (VitaliiDC8) [reply src/components/Paginator/components/PaginatorItem.tsx:24]: it cannot be like this. we use event.preventDefault() on click an tag. App should be not reloaded

[the remark this answers is not in the document]
Comment (niktverd) [reply src/components/Paginator/components/PaginatorItem.tsx:24]: Got it. Issue resolved

File: src/components/Paginator/Paginator.tsx [hunk 2]
```diff
@@ -33,6 +35,9 @@ export const Paginator = ({
         getPagesCount({itemsPerPage, totalItems, maxPages}),
     );
 
+    const {locale} = useContext(LocaleContext);
+    const blogPath = getBlogPath(locale?.pathPrefix || '');
```
Comment (NikitaCG) [inline src/components/Paginator/Paginator.tsx:39]: Now the logic of creating an url is divided into two components. Maybe we should put this code in the Pagination Item ?

Comment (VitaliiDC8) [reply src/components/Paginator/Paginator.tsx:39]: ok, I will do it

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**